### PR TITLE
Implementation of certificates endpoint

### PR DESCRIPTION
**Pull request by omgimanerd**
Hi there, DigitalOcean's API now supports the uploading of SSL certificates. This is a base untested implementation of that feature. I intend to add tests and other necessary support frameworks to this pull request in the near future. Please let me know if there's anything I should change to adhere to your code convention.

I also intend to send a pull request for the new firewall support on DigitalOcean's API. Let me know if this is the direction you want to take for this package.

### CODE REVIEW

**Comment by phillbaker** [comment]
On a quick look, this looks great! I'm going to be away from my computer for a few days, but definitely happy to merge when I get back.

**Comment by phillbaker** [comment]
Hi @omgimanerd just checking in on this, did you have a chance to look at the test coverage for this PR?

**Comment by omgimanerd** [comment]
Not yet, I've been out for the past few days. I'll fix the tests and add more for the certificates endpoint in the next few days.

**Comment by phillbaker** [comment]
Hey @omgimanerd, hope things are going well. Going through and cleaning stuff up. Want me to take a crack at this?

**Comment by omgimanerd** [comment]
Hey there, sorry about my inactivity :( College has been very busy.
I'm not quite sure how tests should be written as I haven't read through your testing infrastructure for this yet.

**Comment by phillbaker** [comment]
No worries! Should be able to base it off of an existing test file, [this PR](https://github.com/phillbaker/digitalocean-node/pull/24) is a good example.

**Comment by omgimanerd** [comment]
Github is squashing in a bunch of other changes here to reconcile the versions because I pulled from upstream to update my code base.

**Comment by phillbaker** [comment]
Awesome! 

Did you hit the "update this branch" button on github.com? What's typically a little cleaner is to `rebase` locally off of the updated changes. Try this:
```
git checkout master
git remote add upstream git@github.com:phillbaker/digitalocean-node.git
git pull upstream/master
git checkout certificates
git rebase master # you may have some conflicts to resolve, but this should remove the duplicate commits
git push -f origin certificates # you'll need to force push your branch since you've rebased it off of master
```

**Comment by omgimanerd** [comment]
Done. I usually don't use rebase because I work on master-only repositories and force pushing is messy, but since I'm pushing this onto my own fork it all works out.

**Comment by phillbaker** [comment]
Awesome thanks, can you also remove the `package-lock.json` file?

File: lib/digitalocean/certificate.js
Comment (phillbaker) [inline lib/digitalocean/certificate.js]: Looks like this link should be `https://developers.digitalocean.com/documentation/v2/#certificates`?

**Review by phillbaker — APPROVED** [review]
Looks good, just need to remove the lock file and update the link!

**Comment by omgimanerd** [comment]
I think I should also add documentation to the README for this. Where should I put it in relation to the other entries under "All resources and actions"?

**Comment by phillbaker** [comment]
Oh, yup, definitely! Want to add it in the readme after the last resource? After https://github.com/phillbaker/digitalocean-node#tag-resource is good.

**Comment by omgimanerd** [comment]
Now that I think about it, shouldn't it make sense to alphabetize the entries in the README, like the way digitalocean does it on their API docs?

**Comment by phillbaker** [comment]
Yup, makes sense, probably good to do as a separate PR!

**Comment by omgimanerd** [comment]
Sounds good, I'll start a new PR. Is there anything else that I need to do or change before you merge this PR?

**Comment by phillbaker** [comment]
Merged, thanks again!

**Comment by phillbaker** [comment]
Will get a new version out later today.

**Pull request merged**
